staging/script.xsqueeze/resources/settings.xml

```diff
@@ -45,4 +45,19 @@
        <setting label="(but if it is too low you may experience crashes crashes)" type="lsep"/>
        <setting id="waitOnAS" type="number" label="Max wait seconds for artist slideshow to exit cleanly... (default 10)" default="10"/>
     </category>
+    <category label="HDMI CEC">
+       <setting label="Please go to http://kodi.wiki/view/Add-on:XSqueeze and read about how to" type="lsep"/>
+       <setting label="install the required dependencies." type="lsep"/>
```

Comment:
yes, it is simple. I will inline the advice for installing trainman's  libcec python binding here. libCEC comes standard with openelec and e.g. rasbian. As long as the maintainers haven´t upgraded to libCEC 3.0 we will need trainman's binding. In a later release we can go without it because the python api will be integrated in libCEC 3.0.